__tests__/app.test.js

```diff
@@ -122,6 +122,69 @@ describe('Topics', () => {
 
 }); // Describe: Topics
 
+describe('Articles', () => {
+    
+    describe('GET /api/articles/:article_id', () => {
+        
+        test('should respond with a 200 status, article object & appropriate properties when provided a valid id', () => {
+            // arrange
+            const articleId = 1;
+            // act
+            return request(app)
+            .get(`/api/articles/${articleId}`)
+            .expect(200)
+            .then((response) => {
+                // assert
+                expect(response.body).toHaveProperty('article');
+                expect(response.body.article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    /**
+                     * TODO: could extend custom matcher: toBeValid<Request|APIEndpoint|...> to accept URL
+                     * + others & use to validate article_img_url, amongst other things.
+                     */
+                    article_img_url: expect.any(String)
```

Comment:
It is a valid test, however, as we are only receiving 1 object back, we can make this airtight by actually testing the values as well rather than expect.any. I would recommend changing this at some point, not urgent!